client/go/outline/config/module.go

```diff
@@ -100,6 +100,19 @@ func NewDefaultTransportProvider(tcpDialer transport.StreamDialer, udpDialer tra
 	// Stream dialers.
 	streamDialers.RegisterSubParser("shadowsocks", NewShadowsocksStreamDialerSubParser(streamEndpoints.Parse))
 
+	// IPTable support (TCP anly).
+	streamDialers.RegisterSubParser("iptable", func(ctx context.Context, input map[string]any) (*Dialer[transport.StreamConn], error) {
+		streamDialer, err := parseIPTableStreamDialer(ctx, input, streamDialers.Parse)
+
+		return &Dialer[transport.StreamConn]{
+			Dial: streamDialer.DialStream,
+			ConnectionProviderInfo: ConnectionProviderInfo{
+				ConnType: ConnTypeTunneled,
+				FirstHop: "",
+			},
+		}, err
+	})
```

Comment:
Let's remove the blank line and keep sorted by name.

```suggestion
	streamDialers.RegisterSubParser("iptable", func(ctx context.Context, input map[string]any) (*Dialer[transport.StreamConn], error) {
		streamDialer, err := parseIPTableStreamDialer(ctx, input, streamDialers.Parse)

		return &Dialer[transport.StreamConn]{
			Dial: streamDialer.DialStream,
			ConnectionProviderInfo: ConnectionProviderInfo{
				ConnType: ConnTypeTunneled,
				FirstHop: "",
			},
		}, err
	})
	streamDialers.RegisterSubParser("shadowsocks", NewShadowsocksStreamDialerSubParser(streamEndpoints.Parse))
```